### PR TITLE
input: Check jsdoc

### LINKED ISSUE #830: input: Check jsdoc

### PR DESCRIPTION
**Pull request by lapinlapin**
Close #830

### CODE REVIEW

File: common.blocks/input/_has-clear/input_has-clear.js [hunk 1]
```diff
@@ -1,5 +1,14 @@
...
+ */
+
 modules.define('input', function(provide, Input) {
 
+/**
+ * @exports
+ * @class input
+ * @bem
+ */
 provide(Input.decl({ modName : 'has-clear', modVal : true }, {
```
Comment (sipayRT) [inline common.blocks/input/_has-clear/input_has-clear.js]: добавь `/** @lends input.prototype */`

**Comment by sipayRT** [comment]
@dfilatov check pls

**Comment by sipayRT** [comment]
@dfilatov ping

**Comment by dfilatov** [comment]
lgtm

**Pull request merged**